src/main/java/org/opensrp/web/rest/LocationTagResource.java

```diff
@@ -0,0 +1,123 @@
+package org.opensrp.web.rest;
+
+import org.joda.time.DateTime;
+import org.joda.time.LocalDate;
+import org.opensrp.domain.LocationTag;
+import org.opensrp.service.LocationTagService;
+import org.opensrp.util.DateTypeConverter;
+import org.opensrp.util.TaskDateTimeTypeConverter;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.dao.DuplicateKeyException;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.google.gson.JsonSyntaxException;
+
+@Controller
+@RequestMapping(value = "/rest/locationTag")
+public class LocationTagResource {
+	
+	private static Logger logger = LoggerFactory.getLogger(LocationTagResource.class.toString());
+	
+	public static Gson gson = new GsonBuilder().registerTypeAdapter(DateTime.class, new TaskDateTimeTypeConverter())
```

Comment:
Use Jackson for serialization.. We are in process of removing gson so that there is only one serialization library used